scripts/RunHPCNodes.py

```diff
@@ -0,0 +1,115 @@
+# Taco Niet
+# Date:  September 26, 2016 - 2018
+# Script to run OSeMOSYS with various parameters using full nodes on a SLURM scheduler managed HPC system.
+
+#!/usr/bin/python
+import subprocess # for running glpsol
+from shutil import copy2 # Allow us to make the needed copies of the parameter file
+from math import sqrt
+import zipfile, os, time, io
+
+if __name__ == '__main__':
+    modelruns = []
+    tailcommands = []
+
+	# Setup task list here.  This can be created by a series of nested for loops.
+    param3List = ['10.6', '15.9', '21.2']
+    param2List = ['0000', '0050', '0100']
+    param1List = ['35', '65', '100']    
+
+    NumRunsTotal = len(param2s)*len(param3Levels)*len(param1s)
+    
+    NumRunstoGroup = 600  # Number of runs to group for each qsub command
+    NumberofCores = 32
+    WallTime = '06:00:00' # Walltime in hh:mm:ss format
+    ZipWallTime = '08:00:00' # Walltime in hh:mm:ss format for zipping files
+    
+    for param1 in param1List:
```

Comment:
You can replace the nested loop with `zip` for example:
```python
for param1, param2, param3 in zip(param1List, param2List, param3List):
    <do stuff>
```

Reply:
Eventually, it might be better to separate creating the sample of parameter values you iterate through from then creating the jobs. You could then use this script with something like SALib to conduct a global sensitivity analysis of a model.

Reply:
I hadn't seen SALib but that looks like a nice tool.  Good suggestion - when I next get to using this script I'll see about adding this in.